DevCycle.SDK.Server.Local.MSTests/DevCycle.SDK.Server.Local.MSTests.csproj

```diff
@@ -5,8 +5,8 @@
     <IsPackable>false</IsPackable>
     <ReleaseVersion>1.0.1</ReleaseVersion>
     <LangVersion>latest</LangVersion>
-    <TargetFrameworks>net6.0;netstandard2.1</TargetFrameworks>
     <OutputType>Library</OutputType>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
had to do this to get the tests runnable in Rider, is this something we want to keep?

Reply:
Yep, it's just a local testing thing.

Reply:
Shouldn't be necessary, I can run the tests without this explicit target. 

Try right clicking the MsTests project and choosing Run Unit Tests ?

Reply:
yeah for some reason it works now after changing that back, it wasn't letting me run using the gutter buttons before..